### ISSUE TITLE
Move to newer Maps control

### ISSUE THREAD

**Issue by dariuszparys** [OpenLiveWriter/OpenLiveWriter#15]
Map experience on Win10 isn't great. Looks like old map controls still in use and should move to newest Map SDK and/or move to OpenStreetMaps.

Initially you get:

![map1](https://cloud.githubusercontent.com/assets/104804/11685389/d8329430-9e78-11e5-860b-b80ef0057d2d.png)

and then after 'allowing blocked content'

![map2](https://cloud.githubusercontent.com/assets/104804/11685390/e223cbee-9e78-11e5-9fc2-2769ca272e97.png)

Making it effectively useless as-is.

@timheuer

**Comment by RobDolin** [comment]
@AdilMughal indicated an interest in working on this

**Comment by adilmughal** [comment]
Yes @RobDolin I am exploring it at the moment. I am considering to drop existing browser control and use the Bing Maps WPF control as mentioned [here](http://blogs.msdn.com/b/rbrundritt/archive/2013/11/08/using-bing-maps-in-winforms.aspx).

**Comment by timheuer** [comment]
@adilmughal please don't bring in WPF...we should be reducing dependencies :-).  One suggestion might be to examine the value the map thing is adding and if OpenStreetMaps might be an option.

**Comment by adilmughal** [comment]
@timheuer by reducing dependencies you mean framework level right? I am not sure I am aware of "reducing dependencies" goal here! 

There is another option [GMap.NET](http://greatmaps.codeplex.com/) which can integrate with OpenStreetMaps as well or we can actually go with browser and APIs which might require relatively more effort.

**Comment by timheuer** [comment]
@adilmughal - there currently is no WPF in the app...if we bring some in, it adds a new dependency (WPF) and interior with Windows Forms (via WPFHost).  It isn't the greatest path in my opinion.  Evaluation of the maps first should be taken into account before just leaping in to replace.  This was my issue I logged based on the experience I had...but anyone working on it should really determine the validity of anything perceived to be better.

**Comment by adilmughal** [comment]
Right, I see your WPF point @timheuer . Actually the first thing that came to my mind was "is this feature actually used by bloggers or do we need to support it or drop it?" Dont get me wrong, Im not saying we remove it but just out of criousity! Do we have information around that? 

Wr can actually do a twitter poll related to "Insert Map" usefulness @shanselman

**Comment by willduff** [comment]
My personal opinion is to remove it and our user community can replace it with a plugin if there is enough need. Maybe @ScottIsAFool already has a maps plugin, actually?

But before we just remove it, I agree with getting some data. We could add some HockeyApp telemetry just to get a count of how many users click on the maps button. Or the twitter poll sounds fine to me as well.

**Comment by hashhar** [comment]
@willduff I was just brushing up through issues for the 0.6 milestone. What is our stand on this? Should we add telemetry or replace it?

**Comment by willduff** [comment]
We haven't added any telemetry yet, so this would be a first. I would be interested in using this as a test of our telemetry system, but we can ship it in any version 0.6.X.X (or later). I don't have a problem pushing this out if needed.

**Comment by adilmughal** [comment]
@willduff 

1) Is there any work done on which telemetry platform to use etc.? 
2) With Telemetry, would we capture event like map used or rather a post published with embedded map? Does this make sense?

**Comment by ScottIsAFool** [comment]
Could we use Application Insights? If so, @martinwoodward, is there an account we should use?

**Comment by martinwoodward** [comment]
HockeyApp is what the Application Insights folks suggested we use for client apps. I'm very keen on telemetry so long as we regularly publish usage data generated by our community.

The point about removing the map control and adding it back via add-in seems like a good idea as well.

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
